rewrite-core/src/main/java/org/openrewrite/style/FormatStyle.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.style;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.With;
+import lombok.experimental.FieldDefaults;
+
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+@Data
+@With
+public class FormatStyle implements Style{
```

Comment:
No need for `@With` and I would rename this to either `TextFormatStyle` or `GeneralFormatStyle` to indicate this style applies for all text-based files.